src/components/home/QuoteBox.tsx

```diff
@@ -1,20 +1,18 @@
 const QuoteBox = ({ quote, name }: { quote: string; name: string }) => {
   return (
     <div className="flex justify-center">
-      <div className="font-lora relative w-3/5">
+      <div className="font-lora relative w-10/12 sm:w-3/5 md:w-3/5">
         <div className="absolute right-4 top-4 h-full w-full rounded-xl bg-csa-gray-200 bg-opacity-50" />
         <div className="relative flex flex-col gap-8 rounded-xl bg-white p-4">
-          <div className="px-[2vw] pt-10">
+          <div className="pt-10">
             <p className="text-8xl font-bold leading-4 text-csa-yellow-100">
               “
             </p>
-            <p className="text-md text-right font-semibold text-csa-gray-100">
+            <p className="text-right text-sm font-semibold text-csa-gray-100">
```

Comment:
```suggestion
            <p className="text-right text-sm md:text-base font-semibold text-csa-gray-100">
```
Sorry, I think I didn't explain what i meant properly. I wanted to say that text-base looks good for larger screen sizes and text-sm looks better for smaller screen sizes. I added a suggestion above for using the md breakpoint to change the text size to text-base on larger screens.